src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js

```diff
@@ -304,7 +312,24 @@ class DatagridCell extends React.Component {
     const modifiers = this.getModifiers();
     const ContentComponent = editable ? 'button' : 'div';
 
-    const content = !['list', 'boolean'].includes(type) && (
+    if (type === 'boolean') {
```

Comment:
I would prefer this change to be a general way to handle "boolean" and then be fix in Input component instead than here.